src/utils/navigate.ts

```diff
@@ -1,18 +0,0 @@
-import { useCallback, MouseEvent } from 'react';
-import { useRouter } from 'next/router';
-
-// navigates to the given url. Opens in new tab if meta key (command on mac) is pressed
-export function useNavigateToUrl() {
```

Comment:
This wasn't used as far as I could tell, and we should really prefer anchor tags rather than this.